### DISCUSSION TITLE
Table of Content Placement

### DISCUSSION THREAD

**Discussion by nurbaitifitriyani** [mecha-cms/mecha#210] [Q&A]
Saya terapkan `[[t-o-c]]` setelah paragraf pertama kok tidak bisa ya?

**Comment by taufik-nurrohman** [comment]
Pakai `mecha-cms/mecha` dan `mecha-cms/x.t-o-c` versi berapa mba? Oya fitur ini kemungkinan besar akan dihapus di Mecha versi 3.0.0. Digantikan dengan metode yang lebih baik.

Tidak bisanya itu dia render jadi kosongan apa masih muncul cetakan `[[t-o-c]]` di konten? Kalau masih muncul cetakan kemungkinan disebabkan karena belum memasang ekstensi `mecha-cms/x.block`.

**Reply by nurbaitifitriyani** [reply]
`mecha-cms/mecha` versi `2.6.4` dan `mecha-cms/x.t-o-c` versi `2.5.6`. Saya selalu meng-*update* Mecha dan ekstensinya versi stabilnya sih, Mas. Saya mengharapkan ada suatu cara untuk menempatkan TOC sesuai kehendak pengguna tanpa mengganggu konten postingan, ya cukup YAML saja untuk identitas fail (judul, deskripsi dan penulis), selebihnya ya konten.

> Kalau masih muncul cetakan kemungkinan disebabkan karena belum memasang ekstensi `mecha-cms/x.block`.

Ya, Mas. Ternyata saya belum pasang ekstensi `mecha-cms/x.block`. Setelah saya pasang dan tes kembali, berjalan.

**Reply by taufik-nurrohman** [reply]
> Saya mengharapkan ada suatu cara untuk menempatkan TOC sesuai kehendak pengguna tanpa mengganggu konten postingan.

Rencananya Saya mau tambahkan properti laman dinamis seperti halnya `$page->tags` di ekstensi `tag` supaya pengguna bisa menaruh markup tabel konten secara bebas. Contoh kasus adalah ketika pengguna ingin menampilkan markup tabel konten di bilah sisi (ini berlaku juga untuk ekstensi `next`).

Properti `$page->content` nantinya akan polos saja tidak berubah susunannya.
